@here/harp-olp-utils/package.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
Again it's not my decision.
@ninok, @musculman Shall we provide OLP specific packages in harp.gl ?

Reply:
yeah that's fine. We will also move our OLP specific datasources to github at some point (hopefully soon)